WNPRC_EHR/resources/queries/study/ultrasound_measurements.js

```diff
@@ -0,0 +1,19 @@
+var WNPRC = require("wnprc_ehr/WNPRC").WNPRC;
+require("ehr/triggers").initScript(this);
+
+function onInit(event, helper){
+    if (event === "update") {
+        helper.setScriptOptions({
+            allowDatesInDistantPast: true
+        });
+    }
+}
+
+function onInsert(helper, scriptErrors, row, oldRow){
+}
+
+function onUpdate(helper, scriptErrors, row, oldRow){
+}
+
+function onUpdate(helper, scriptErrors, row, oldRow){
+}
```

Comment:
I don't think we need a description here because all of this info is stored in the research_ultrasounds dataset as well and is recorded in the description there. This table is mostly for clean data storage of the individual measurements